install_helper.sh

```diff
@@ -7,8 +7,8 @@ get_releases() {
 }
 
 linux_install() {
-  bin_path=$HOME/.local/bin
-  lib_path=$HOME/.local/lib
+  bin_path=$HOME/.local/share/code-server/bin
+  lib_path=$HOME/.local/share/code-server/lib
```

Comment:
I don't think we need a subdirectory of lib here, let's just store the versions in `share/code-server/<version_number>`

Reply:
Then we can remove the "backup" step as well, correct?

Reply:
On second thought, let's keep it there in case someone reinstalls

Reply:
Yea we can, we just keep all versions but replace the symlink to the binary in bin.